www/docs/en/dev/config_ref/index.md

```diff
@@ -324,6 +324,7 @@ OverrideUserAgent(string) <br/> ==Android== ==iOS== | If set, the value will rep
 PageLength(float) <br/> ==iOS== | *Default: 0* <br/>  The size of each page, in points, in the direction that the pages flow. When PaginationMode is right to left or left to right, this property represents the width of each page. When PaginationMode is topToBottom or bottomToTop, this property represents the height of each page. The default value is 0, which means the layout uses the size of the viewport to determine the dimensions of the page.
 PaginationBreakingMode(string) <br/> ==iOS== | *Default: page* <br/> Allowed values: page, column <br/>  Valid values are page and column.The manner in which column- or page-breaking occurs. This property determines whether certain CSS properties regarding column- and page-breaking are honored or ignored. When this property is set to column, the content respects the CSS properties related to column-breaking in place of page-breaking.
 PaginationMode(string) <br/> ==iOS== | *Default: unpaginated* <br/> Allowed values: unpaginated, leftToRight, topToBottom, bottomToTop, rightToLeft <br/>  This property determines whether content in the web view is broken up into pages that fill the view one screen at a time,or shown as one long scrolling view. If set to a paginated form, this property toggles a paginated layout on the content, causing the web view to use the values of PageLength and GapBetweenPages to relayout its content.
+PreferredContentMode(string) <br/> ==iOS== | *Default: auto* <br/>  Sets the content mode (user agent) for the webview and inappbrowsers webview on iPads. Valid values are: `mobile` and `desktop`.
```

Comment:
Personally I'd style it as `WebView` and `InAppBrowser`, but the docs are already mixed.

Reply:
```suggestion
PreferredContentMode(string) <br/> ==iOS== | *Default: auto* <br/>  Sets the content mode (user agent) for the WebView and InAppBrowsers WebView on iPads. Valid values are: `mobile` and `desktop`.
```